types/better-sqlite3/index.d.ts

```diff
@@ -62,8 +62,17 @@ declare namespace BetterSqlite3 {
         transaction<F extends VariableArgFunction>(fn: F): Transaction<F>;
         exec(source: string): this;
         pragma(source: string, options?: Database.PragmaOptions): unknown;
-        function(name: string, cb: (...params: unknown[]) => unknown): this;
-        function(name: string, options: Database.RegistrationOptions, cb: (...params: unknown[]) => unknown): this;
+        // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
+        function<Parameters extends unknown[] = unknown[], Result = unknown>(
+            name: string,
+            cb: (...params: Parameters) => Result,
```

Comment:
I don't really understand why this is an improvement; anything that can be passed in here should be passable to `(...params: unknown[]) => unknown`, no?

Reply:
See https://tsplay.dev/mA9P4m.

I think it can only be resolved by adding generics.

Reply:
Why not just use `(...args: any[]) => any`? I don't see any way that this function's result can be used in any way, so besides "it's a function", I don't think being exact is very useful.

Reply:
Fixed.